librosa/core/spectrum.py

```diff
@@ -946,8 +955,8 @@ def amplitude_to_db(S, ref=1.0, amin=1e-5, top_db=80.0):
     else:
         ref_value = np.abs(ref)
 
-    magnitude **= 2
-    return power_to_db(magnitude, ref=ref_value**2, amin=amin**2,
+    power = magnitude**2
```

Comment:
I get the mutation was to avoid a copy, _but_ for newbs like me it's a lot clearer with the name `power` because when I was new to LibROSA's code base I conflated magnitudes and power, _a lot_.